src/main/java/jenkins/bouncycastle/api/PEMEncodable.java

```diff
@@ -487,5 +491,25 @@ private static String encodeBase64(@Nonnull byte[] data) {
         return Base64.decode(data);
     }
 
+    /**
+     * Registers bouncy castle on the slave JVM
+     * 
+     * @param channel to the slave
+     * @return result of the operation
+     * @throws Exception if there is a problem registering bouncycastle
+     */
+    public static boolean registerBCOnSlave(@Nonnull Channel channel) throws Exception {
```

Comment:
Hmmm I though you were putting this in a sensible utility class. What has this to do with registering BC.

Perhaps the callable should not be @Restricted, rather have a private constructor and put the utility method on the class directly